src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginDiscoverer.cs

```diff
@@ -63,6 +63,17 @@ internal class TestPluginDiscoverer
 
 #if !WINDOWS_UAP
 
+            if(!extensionPaths.Any())
```

Comment:
Is this code supposed to be in `!WINDOWS_UAP`?

Reply:
the other code has no significance, since we are no longer compiling for UAP, we can remove all other code.

Reply:
Please remove it.